go/arrow/internal/testing/types/extension_types.go

```diff
@@ -18,20 +18,172 @@
 package types
 
 import (
+	"bytes"
 	"encoding/binary"
 	"fmt"
 	"reflect"
+	"strings"
+
+	"github.com/goccy/go-json"
 
 	"github.com/apache/arrow/go/v12/arrow"
 	"github.com/apache/arrow/go/v12/arrow/array"
+	"github.com/google/uuid"
 	"golang.org/x/xerrors"
 )
 
+type UUIDBuilder struct {
+	*array.ExtensionBuilder
+	dtype *UUIDType
```

Comment:
the failures are because this field is unused here

Reply:
Oh ok. Fixed. I wasn't able to find it from the action. Where should I look for it next time? (Can I run those linters locally as well?)

Reply:
So that was actually found by the Go build failing, and I saw it by looking at the log file for the failing Go workflows.

You can also run the linters (or most of the workflows) by following the instructions here: https://arrow.apache.org/docs/developers/continuous_integration/archery.html to install the `archery` utility in the repo and then running `archery docker run <job>` as described [here](https://arrow.apache.org/docs/developers/continuous_integration/docker.html).

Reply:
Sounds good. Thanks, I'll try those for next PR! Tests passing now.